components/side-drawer/src/vwc-side-drawer-base.ts

```diff
@@ -93,32 +108,38 @@ export class VWCSideDrawerBase extends LitElement {
 		const topBar = this.hasTopBar ? this.renderTopBar() : '';
 		const scrim = (this.type === 'modal' && this.open) ? this.renderScrim() : '';
 		const alternate = this.alternate ? 'vvd-scheme-alternate' : undefined;
+		const start = this.position === 'start';
+		const end = this.position === 'end';
 
 		const classes = {
 			'side-drawer--alternate': this.alternate,
 			'side-drawer--dismissible': dismissible,
 			'side-drawer--modal': modal,
 			'side-drawer--open': this.open,
+			'side-drawer--start': start,
+			'side-drawer--end': end,
+		};
+
+		const contentClassed = {
+			'side-drawer--start': start,
```

Comment:
maybe we can change the class to start & end without the side-drawer because it's a bit strange on the content to use `side-drawer--end`

Reply:
I want to maintain consistency with other side-drawer classes. I think we should remove the "side-drawer--" prefix from all classes, but then it may break for some users.  @yinonov what do you think?

Reply:
It's implementation details not API so it won't break for authors.
But it's our convention to have it prefixed with the scope. Also don't use double hyphens

Reply:
You're right, I was confused with the --side-drawer-background-color